src/canopy_ncf_io_mod.F90

```diff
@@ -369,6 +369,30 @@ SUBROUTINE canopy_outncf_init
         c_canheight%iend(1) = nlon
         c_canheight%iend(2) = nlat
 
+        c_dh%fld = fillreal
+        c_dh%fldname = 'dh'
+        c_dh%long_name = 'zero-plane displacement height (z/h)'
```

Comment:
@angehung5 This is actually the ratio of zero plane displacement height to canopy height, d/h.
```suggestion
        c_dh%long_name = 'ratio of zero-plane displacement height to canopy height (d/h)'
```